### PR TITLE
feat(ADBP-2588) - Removed navigation on changing payment method [tran…

### PR DESCRIPTION
**Pull request by hiteshdigit101**
ADBP-2588 Implemented

### CODE REVIEW

File: src/components/transfer-out-component/index.tsx [hunk 1]
```diff
@@ -214,135 +205,120 @@ const TransferOutComponent = ({
...
-                        }
-                      }}
+        <Formik
+          innerRef={formikRef}
+          enableReinitialize={true}
+          initialValues={NewRecipientData.empty()}
+          validationSchema={AccountNumberSchema}
+          onSubmit={(values) => {
+            const { type, number } = values;
+            console.log("Number --> ", number);
```
Comment (sharmal33) [inline src/components/transfer-out-component/index.tsx:246]: remove log

File: src/components/transfer-out-component/style.ts [hunk 2]
```diff
@@ -1,64 +1,65 @@
...
       borderRadius: 30,
-      alignSelf: 'center',
+      alignSelf: "center",
       marginTop: 13,
-      flexDirection: 'row',
+      flexDirection: "row",
     },
     setButtonText: {
-      color: '#FCFCFC',
+      color: "#FCFCFC",
```
Comment (sharmal33) [inline src/components/transfer-out-component/style.ts:51]: use theme colors

File: src/components/transfer-out-component/style.ts [hunk 3]
```diff
@@ -68,30 +69,30 @@ const useMergeStyles = (style?: TransferOutComponentStyles): TransferOutComponen
       paddingVertical: 10,
     },
     title: {
```
Comment (sharmal33) [inline src/components/transfer-out-component/style.ts:71]: use theme colors

File: src/components/transfer-out-component/style.ts [hunk 4]
```diff
@@ -102,51 +103,51 @@ const useMergeStyles = (style?: TransferOutComponentStyles): TransferOutComponen
...
     },
     alertIcon: {
       height: 75,
       width: 75,
-      alignSelf: 'center',
+      alignSelf: "center",
     },
     alertHeading: {
-      color: '#000000',
+      color: "#000000",
```
Comment (sharmal33) [inline src/components/transfer-out-component/style.ts:118]: use theme colors

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/transfer-out-component/style.ts (hunk 2 above)
Comment (hiteshdigit101) [reply src/components/transfer-out-component/style.ts]: removed log

File: src/components/transfer-out-component/style.ts (hunk 4 above)
Comment (hiteshdigit101) [reply src/components/transfer-out-component/style.ts]: changed to theme colors

File: src/components/transfer-out-component/style.ts (hunk 3 above)
Comment (hiteshdigit101) [reply src/components/transfer-out-component/style.ts:71]: changed to theme colors

File: src/components/proxy-transfer-component/index.tsx [hunk 5]
```diff
@@ -160,104 +134,95 @@ const ProxyTransferComponent: React.FC<IProxyTransferComponent> = (
...
+          keyboardType={keyboardType}
+          onChangeText={(e) => onChangeProxyTypeValue(e)}
         />
-      </ScrollView>
+        {!isProxyValid && (
+          <View style={innerStyles.errorTextContainerStyle}>
+            <DangerIcon />
+            <Text
+              style={[innerStyles.errorTextStyle, { color: colors.secondaryButtonLabelColor }]}
+            >{`Please enter a valid ${proxyName.toLowerCase()}.`}</Text>
```
Comment (catluong101digital) [inline src/components/proxy-transfer-component/index.tsx:186]: can you use it from translation instead

File: src/components/proxy-transfer-component/index.tsx [hunk 6]
```diff
@@ -306,12 +270,10 @@ const innerStyles = StyleSheet.create({
   },
   copyContainerText: {
     fontSize: 12,
-    color: "#1b1b1b",
     marginHorizontal: 10,
     fontFamily: "Poppins",
```
Comment (catluong101digital) [inline src/components/proxy-transfer-component/index.tsx:274]: use font from theme component

File: src/components/select-transfer-type-component/index.tsx [hunk 7]
```diff
@@ -46,72 +50,85 @@ export type ItemsList = {
...
         </View>
-        {/* Button */}
-        {/* {isButtonVisible ? (
-          <TouchableOpacity onPress={() => onPressSet(paymentType)} style={styles.setButton}>
-            <Text style={styles.setButtonText}>{buttonTitle}</Text>
-          </TouchableOpacity>
-        ) : null} */}
-      </View>
+
+        {paymentType?.Name !== "AccountNumber" && (
```
Comment (catluong101digital) [inline src/components/select-transfer-type-component/index.tsx:105]: can we move this string "AccountNumber" to a constant file

File: src/components/transfer-out-component/index.tsx [hunk 8]
```diff
@@ -214,135 +205,111 @@ const TransferOutComponent = ({
...
+          onSubmit={(values) => {
+            const { type, number } = values;
+            onSearch(number);
+          }}
+        >
+          {({ values }) => {
+            setAccountNumber(values?.number);
+            return (
+              <>
+                <Text style={styles.labelTextStyle}>{"Account number"}</Text>
```
Comment (catluong101digital) [inline src/components/transfer-out-component/index.tsx:253]: use text from translation

File: src/components/transfer-out-component/style.ts [hunk 9]
```diff
@@ -49,16 +52,14 @@ const useMergeStyles = (style?: TransferOutComponentStyles): TransferOutComponen
     setButtonText: {
       color: '#FCFCFC',
```
Comment (catluong101digital) [inline src/components/transfer-out-component/style.ts:53]: use color from theme component

File: src/components/select-transfer-type-component/index.tsx [hunk 10]
```diff
@@ -46,72 +50,85 @@ export type ItemsList = {
...
-  const { getPaymentMethod,paymentMethods } =
-    useContext(TransferContext);
-  const {bankWallet} = useContext(WalletContext)
+  const { getPaymentMethod, paymentMethods } = useContext(TransferContext);
+  const { bankWallet } = useContext(WalletContext);
   const { i18n } = useContext(ThemeContext);
-  const [paymentType, setPaymentType] = useState<PaymentMethod>();
+  const [paymentType, setPaymentType] = useState<PaymentMethod | null>(null);
+  const defaultPaymentMethod = useMemo(
+    () => paymentMethods.find((item) => item?.Name === "MobileNumber") || null,
```
Comment (catluong101digital) [inline src/components/select-transfer-type-component/index.tsx:67]: can we move this to a constant file

**Review by catluong101digital — APPROVED** [review]

**Review by sharmal33 — APPROVED** [review]

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**